yarn-error.log

```diff
@@ -0,0 +1,51 @@
+Arguments: 
+  /usr/bin/node /usr/bin/yarn lint:flow
```

Comment:
We shouldn't be committing this file!

Reply:
I added *.log to .gitignore, but you'll want to delete this file.